doc/latex.rst

```diff
@@ -1024,6 +1024,14 @@ Environments
 Miscellany
 ~~~~~~~~~~
 
+- Every text paragraph in document body starts with `\sphinxAtStartPar`.
+  Currently, this is used to insert a zero width horizontal skip which
+  is a trick to allow TeX hyphenation of the first word of a paragraph
+  in a narrow context (like a table cell). For ``'lualatex'`` which
+  does not need the trick, the `\sphinxAtStartPar` does nothing.
+
+  .. versionadded:: 3.5.0
```

Comment:
To the best of my knowledge the change can not break PDF generation. As far as I can tell, at worst it could add an extra blank line if the first child is implemented via a "vertical" environment: typically a list, but this seems excluded: in my understanding a list can not be first child of a paragraph node. An image is not a problem if it is not embedded into "figure" (for example I image inserted from a substitution). I tested that math like `equation` or `align` do not seem to be affected. At any rate there is no way I can see this can break PDF build whatever nodes follow. The only thing it can cause is this extra blank line. But I can not construct an example from rst source creating this.

Reply:
>To the best of my knowledge the change can not break PDF generation. 

Okay. I understand and agreed this does not break PDF generation. Let us see what will be reported.

>(for example I image inserted from a substitution)

Fortunately, reST's substitution only allows inline elements. Nobody can't insert non "inline" elements under the paragraph node without hacks (by raw node or extensions).

>A substitution definition block contains an embedded inline-compatible directive (without the leading ".. "), such as "image" or "replace".
>https://docutils.sourceforge.io/docs/ref/rst/restructuredtext.html#substitution-definitions